source/module_hamilt_lcao/module_hcontainer/test/demo1_SR.cpp

```diff
@@ -0,0 +1,42 @@
+#include "../hcontainer.h"
+void construct_SR(const Parallel_Orbitals* paraV, const UnitCell& ucell)
```

Comment:
How would you test the demos?

Reply:
I will add UT for demos in module_hcontainer/test/ , these demos will be developed with TDD(Test Driven Development). They are only sseudocode for demonstration at now